youtube_dl/extractor/generic.py

```diff
@@ -1869,6 +1904,7 @@ def _playlist_from_matches(matches, getter=None, ie=None):
         # Look for Brightcove New Studio embeds
         bc_urls = BrightcoveNewIE._extract_urls(webpage)
         if bc_urls:
+            self.to_screen('Brightcove (new) video detected.')
```

Comment:
The other case was some legacy code. I don't see much sense in such messages since it's clear what extractor is delegated to since all messages from the final extractor are prefixed with `IE_NAME`.